cypress/integration/unit/helpers/shift.helper.spec.ts

```diff
@@ -0,0 +1,41 @@
+/// <reference types="cypress" />
+import { DataRow } from "../../../../src/logic/schedule-logic/data-row";
+import { ShiftHelper } from "../../../../src/helpers/shifts.helper"
+
+type TestCase = { arr: DataRow[]; day: number; exp: number };
+
+const testData = [
+    new DataRow("", ["R","DN","W"]),
+    new DataRow("", ["P","D","W"]),
+    new DataRow("", ["D","W","W"])
+    ]
+
+var testCases: TestCase[] = [
```

Comment:
W projekcie używamy "let" dla zmiennych i "const" dla stałych. Tutaj linter powinien to wykryć niestety jest on źle ustawiony i nie obejmuje plików z testami. Wprowadź tutaj tę zmianę, a ja wprowadzę zmiany do lintera w osobnym PR, żeby nie było już takich problemów w przyszłości.